rubies/ruby/CVE-2019-16255.yml

```diff
@@ -1,8 +1,8 @@
 ---
 engine: ruby
 cve: 2019-16255
-url: https://www.ruby-lang.org/en/news/2019/10/01/code-injection-shell-test-cve-2019-16255/
-title: A code injection vulnerability of Shell#[] and Shell#test
+url: https://www.ruby-lang.org/en/news/2019/10/01/code-injection-shell-test-cve-2019-16255
+title: 'A code injection vulnerability of Shell#[] and Shell#test'
```

Comment:
It doesn't appear that single-quotes are needed here:

```ruby
>> YAML.load("title: A code injection vulnerability of Shell#[] and Shell#tes
t\n")
=> {"title"=>"A code injection vulnerability of Shell#[] and Shell#test"}
```

Reply:
will fix ; worried about []